common/src/leap/soledad/common/tests/util.py

```diff
@@ -362,8 +367,14 @@ def start(self):
 %s""" % (
                     self.process.returncode, got_stdout, got_stderr))
             time.sleep(0.01)
+            if (time.time() - boot_time) > self.BOOT_TIMEOUT_SECONDS:
+                self.stop()
+                raise Exception("Timeout starting couch")
```

Comment:
It will only enter this loop if logPath doesn't exists and there is no code inside the while that creates it. This is the reason that I added this check also to this part, but you're right: We need to check those exceptions and call stop or retry, because couchdb process will be orphaned here. :+1: Fixing...